anet-dictionary.yml

```diff
@@ -198,6 +198,7 @@ fields:
     parentTask:
       label: Parent task
       placeholder: Start typing to search for a higher level task
+    childrenTasks: Sub tasks
```

Comment:
Maybe we should stick with the terminology here, since (in our dictionary) top-level tasks are called "Objectives" and sub-level tasks are called "Efforts"; so perhaps something like "(Sub) efforts" (because they will never be top-level tasks)?
That implies that the label for `parentTask` directly above could be something like "Parent objective / effort", also in keeping with the terminology.

Reply:
To be strict with terminology is important.

Yet "Objectives" and sub-level tasks are called "Efforts" can be seen as obsolete, as we step away from this 2 level hard coded model. In principle everything can be renamed to task.